_benchmark/reference/summary-report.md

```diff
@@ -0,0 +1,260 @@
+
+The total number of documents ingested by all nodes within the cluster, over the duration of the race.
+
+## Total Ingest Pipeline time
+
+**Corresponding metrics key**: `ingest_pipeline_cluster_time`
+
+The total time, in milliseconds, spent preprocessing ingest documents by all nodes within the cluster, over the duration of the race.
+
+
+## Total Ingest Pipeline failed
+
+**Corresponding metrics key**: `ingest_pipeline_cluster_failed`
+
+The total number of failed ingest operations by all nodes within the cluster, over the duration of the race.
+
+
+## Throughput
+
+**Corresponding metrics key**: `throughput`
+
+ reports the minimum, mean, median and maximum throughput for each task.
+
+The number of operations that OpenSearch can perform within a certain time period,  per second. The report includes the minimum, mean, median and maximum throughput for each task.
+
+
+## Latency
+
+**Corresponding metrics key**: `latency`
+
+The time period between submission of a request and receiving the complete response. It also includes the wait time the request spends waiting until it is ready to be serviced by OpenSearch. OpenSearch reports several percentile numbers for each task. Which percentiles are shown depends on how many requests OpenSearch can capture during the latency period.
+
+
+## Service time
+
+**Corresponding metrics key**: `service_time`
+
+The time period between sending a request and receiving the corresponding response. This metric can easily be mixed up with latency but does not include waiting time. This is what most load testing tools refer to as “latency” (although it is incorrect). OpenSearch reports several percentile numbers for each task. Which percentiles are shown depends on how many requests OpenSearch can capture during the latency period.
+
+
+
+## Processing time
+
+Processing time is only reported if the setting `output.processingtime` is set to true in the OpenSearch Benchmark configuration file.
+{: note.}
+
+**Corresponding metrics key**: `processing_time`
+
+
+The time period between start of request processing and receiving the complete response. Unlike to service time, this metric also includes OpenSearch’s client side processing overhead. Large differences between service time and processing time indicate a high overhead in the client and can thus point to a potential client-side bottleneck which requires investigation.
+
+
+## Error rate
+
+Corresponding metrics key: `service_time`. Each `service_time` record has a meta.success flag. 
+
+The ratio of erroneous responses relative to the total number of responses. Any exception thrown by the Python OpenSearch client is considered erroneous, for example, HTTP response codes 4xx, 5xx, or network errors (network unreachable). Usually any error rate greater than zero was found by the Alerting plugin. You should investigate the root cause by inspecting OpenSearch and OpenSearch Benchmark logs and rerun the benchmark.
+
```

Comment:
Delete extra line

Reply:
Rewrite "Usually any error rate greater than zero was found by the Alerting plugin." It's an incomplete thought.